lib/src/main/java/com/diffplug/spotless/sql/dbeaver/SQLTokenizedFormatter.java

```diff
@@ -21,11 +21,15 @@
 import java.util.Objects;
 
 /**
+ * **Warning:** Use this class at your own risk. It is an implementation detail and is not
+ * guaranteed to exist in future versions.
```

Comment:
Any sort of annotation we could put on this like `@Internal` or similar?

Reply:
Yes, I think I could easily make up an annotation with the name of `@Internal`. Unless you have any objections @nedtwigg, I'll introduce such an annotation and annotate all necessarily-public implementation detail classes with it.

Reply:
Sounds good!